tools/clean.sh

```diff
@@ -42,9 +42,15 @@ make clean
 cd $SRCDIR/tc/sgx/trusted_worker_manager/enclave
 rm -rf build deps
 
+cd $SRCDIR/examples/enclave_manager/tcf_enclave_manager
+rm -rf tcf_enclave.py tcf_enclave_wrap.cpp
```

Comment:
These files are removed in clean section in Makefile in examples/enclave_manager. I think its better to invoke make clean

Reply:
done as 1a0efa538e05b07b64280f3afd7a13a1cc58ad06

Reply:
this work has been moved into separate PR https://github.com/hyperledger-labs/trusted-compute-framework/pull/42